aws_lambda_powertools/utilities/batch/sqs.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+"""
+Batch SQS utilities
+"""
+
+from typing import List, Optional
+
+import boto3
+from botocore.config import Config
+
+from aws_lambda_powertools.middleware_factory import lambda_handler_decorator
+
+from .base import BasePartialProcessor
+
+
+class PartialSQSProcessor(BasePartialProcessor):
+    def __init__(self, config: Optional[Config] = None):
+        config = config or Config()
+        self.client = boto3.client("sqs", config=config)
+        self.success_messages: List = []
+        self.fail_messages: List = []
+
+        super().__init__()
+
+    def get_queue_url(self):
```

Comment:
```suggestion
    def _get_queue_url(self) -> str:
```

Other than a unit test there is no need to expose this method right?